tests/phpunit/elementor/includes/widgets/test-menu-anchor.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Elementor\Tests\Phpunit\Includes\Widgets;
+
+use Elementor\Plugin;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+class Elementor_Test_Element_Base extends Elementor_Test_Base {
+	const MOCK = [
+		'id' => 'e8e55a1',
+		'elType' => 'widget',
+		'settings' => [ 'anchor' => 'test-anchor' ],
+		'widgetType' => 'menu-anchor',
+	];
+
+	public function test_on_save__with_invalid_data() {
```

Comment:
You're actually testing `get_data_for_save()`

Reply:
```suggestion
	public function test_get_data_for_save__with_invalid_data() {
```

Reply:
The method `get_data_for_save` triggers the protected `on_save` method.